cmd/virt-api/virt-api.go

```diff
@@ -127,6 +127,14 @@ func (app *virtAPIApp) Run() {
 		ApiPath:         "/swaggerapi",
 		SwaggerPath:     "/swagger-ui/",
 		SwaggerFilePath: app.SwaggerUI,
+		Info: swagger.Info{
```

Comment:
Even having this here, it doesn't appear in final json file: 
```json
{
  "swaggerVersion": "1.2",
  "apiVersion": "",
  "basePath": "http://localhost:8183",
  "resourcePath": "/apis/kubevirt.io/v1alpha1",
  "info": {
   "title": "",
   "description": ""
  },
```

Reply:
It is there,  when you look at `/swaggerapi` (the base path). You won't see it under `/swaggerapi/apis/kubevirt.io/v1alpha1`. I hope that this is good enough, if you use the /swaggerapi endpoint as the entrypoint for swagger-gen.